blog/2018/12/07/compound-selectors/index.markdown

```diff
@@ -0,0 +1,210 @@
+---
+status: published
+title: Compound Selectors are Easier than Regexes
+author: brian d foy
+tags:
+    - promises
+    - advent
+images:
+  banner:
+    src: 'banner.jpg'
```

Comment:
Quick note, and I'll add this to the readme. Maybe it should even be a bug for @preaction on Statocles. It seems that the banner and any urls before the fold need to be absolute. That's because the article is also shown on pages like the blog scroll and the tag scrolls and in those cases point urls relative to that page, which doesn't work.

@preaction maybe could statocles fix that up during build?

Reply:
I could add a method on Statocles::Page::ListItem that would fix relative URLs into absolute ones that then the theme could use (https://github.com/MojoliciousDotIO/mojolicious.io/blob/master/theme/blog/index.html.ep#L35). I'll add that to the list o' things.